lms/djangoapps/django_comment_client/permissions.py

```diff
@@ -69,17 +85,16 @@ def check_team_member(user, content):
             cache_key = "django_comment_client.check_team_member.{}.{}".format(user.id, commentable_id)
             if cache_key in request_cache_dict:
                 return request_cache_dict[cache_key]
-            team = CourseTeam.objects.get(discussion_topic_id=commentable_id)
-            passes_condition = team.users.filter(id=user.id).count() > 0
+            team = commentable_team(commentable_id)
+            if team is None:
+                passes_condition = True
```

Comment:
I made it purposefully generic because passes_condition is also true when the commentable is not linked with a team (that's the "not applicable" part of the condition).